dns/ddclient/src/opnsense/mvc/app/models/OPNsense/DynDNS/DynDNS.xml

```diff
@@ -67,6 +67,7 @@
                         <he-net-tunnel>HE.net TunnelBroker</he-net-tunnel>
                         <hetzner>Hetzner DNS Console</hetzner>
                         <inwx>INWX</inwx>
+                        <inwx_native>INWX (native)</inwx_native>
```

Comment:
To some degree this inwx/inwx_native collision is why I'd like to split the ddclient plugins into two separate ones for each backend in the long run.

Reply:
I'm not quite sure, what you mean.
Do you want me to change something?

In the aftermath, I think `INWX (JsonRPC)` would be a better name than `native`, yes?

Reply:
INWX lives in both backends now. Ideally it should be the same entry with the same field supported to avoid confusion over what implementation can do what feature.

This is a very different problem from your PR that just tries to do the right thing. Ideally we should have a separate plugin for ddclient and the native Python backend then this clash of "same same but different" doesn't happen.

Reply:
Well, instead of taking things away for others using the same service name doesn't conflict. We'd just have to aim for feature parity on the additional options provided.